### PR TITLE
Fixed Issues with searchable and searchable activity

### PR DESCRIPTION
**Pull request by Manchips**
Used the wrong program to edit the files resulting in some wonkey errors these should be okay let me know if they don't work though!

### CODE REVIEW

**Comment by ae-ramirez** [comment]
Do these changes compile on your end?
On my end I only end up with errors:
`ParseError at [row,col]:[3,10]
Message: The processing instruction target matching "[xX][mM][lL]" is not allowed.`

**Comment by Youngphil5** [comment]
Just delete the xml and create a new searchable xml file under xml

File: app/src/main/res/xml/searchable.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+    <?xml version="1.0" encoding="utf-8"?>
```
Comment (Youngphil5) [inline app/src/main/res/xml/searchable.xml:3]: hey bro, this line of code is causing errors

Comment (Youngphil5) [reply app/src/main/res/xml/searchable.xml:3]: I think this line of code should be in this file just once. You have it multiple times in this file. (** DUPLICATE CODE**)

File: app/src/main/res/xml/searchable.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Youngphil5) [inline app/src/main/res/xml/searchable.xml:1]: ** DUPLICATE CODE**

**Review by Youngphil5 — CHANGES_REQUESTED** [review]
Hey bro, the searchable class is left empty so it's causing a lot of errors. Plus, in the serchable.xml file, you have this line of code repeated "<?xml version="1.0" encoding="utf-8"?>" on line 1 and line 3. it is causing errors too.

**Pull request merged**